capa/rules.py

```diff
@@ -866,7 +866,8 @@ def _get_rules_for_scope(rules, scope):
         given a collection of rules, collect the rules that are needed at the given scope.
         these rules are ordered topologically.
 
-        don't include "lib" rules, unless they are dependencies of other rules.
+        don't include auto-generated "subscope"/"lib" rules.
```

Comment:
```suggestion
        don't include auto-generated "subscope" rules.
```

Reply:
oh man i like this suggestion feature

Reply:
Yes, very cool! Thanks for the feedback.